src/mmw/js/shim/leaflet.utfgrid.js

```diff
@@ -0,0 +1,359 @@
+L.Util.ajax = function (url, cb) {
```

Comment:
Since we're shimming this in, we should add the license here. https://github.com/danzel/Leaflet.utfgrid/blob/master/MIT-LICENCE.txt